exercises/E-conditionals/exercise.js

```diff
@@ -8,7 +8,11 @@
 
 let name = "Daniel";
 let danielsRole = "mentor";
-
+ if (danielsRole) {
```

Comment:
If I change the  variable called danielsRole to "Student" the code will still return "Hi, I'm Daniel, I'm a mentor"

Reply:
if (danielsRole == "mentor"){return "Hi, I'm Daniel, I'm a mentor"}else{return "Hi, I'm Daniel, I'm a student"}